app/__mocks__/styleMock.js

```diff
@@ -1 +1,6 @@
+<<<<<<< HEAD
+module.exports = {};
+
+=======
 module.exports = {}
+>>>>>>> 74bd68e76ccaca8bc778d8673a6ce90441959a1d
```

Comment:
Whoops looks like a merge conflict got in. Fix here:
```suggestion
module.exports = {};
```